01-bubble-sort/src/components/board/Board.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import { Cell } from '../Cell/Cell';
+import style from './Board.module.scss';
+
+export function Board({ array }: { array: number[] }): JSX.Element {
+  return (
+    <>
+      <div className={style.board}>
+        {array.map((item: number) => {
+          return <Cell key={`cell${item}`} size={item} />;

```

Comment:
Guess, you can have the same keys. The values in the array can be the same.

Reply:
I am creating an array with unique elements